# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 47dc3e2e467e
# Full Hash: 47dc3e2e467e5b5873e327efaab2bf1f872de334
# Author: Paul Adenot <paul@paul.cx>
# Date: 2022-03-08 22:01:59
# Description:
#   Bug 1757618 - Use smaller ring buffer on anything but m1 macs. r=media-playback-reviewers,alwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D140218
# ==============================================================================

diff -r 2c2c3c960a9b -r 47dc3e2e467e modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Mar 08 05:31:12 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Mar 08 07:44:36 2022 +0000
@@ -9579,7 +9579,11 @@
 # reached. The total size of the ring-buffer is slightly bigger than this.
 - name: media.audio.audiosink.threshold_ms
   type: AtomicFloat
+#if defined(XP_MACOSX) && defined(__aarch64__)
   value: 1000.0
+#else
+  value: 200.0
+#endif
   mirror: always
 
 
